apps/els_lsp/src/els_indexing.erl

```diff
@@ -112,6 +113,17 @@ index_references(#{uri := Uri} = Document, 'deep', false) ->
 index_references(_Document, 'shallow', _) ->
   ok.
 
+-spec maybe_start() -> true | false.
+maybe_start() ->
+  case els_config:get(indexing_enabled) =:= false of
```

Comment:
It may be more readable written as:
```
case els_config:get(indexing_enabled) of
  false ->
    ?LOG_INFO("Skipping Indexing (disabled via InitOptions)");
  X when X =:= true; X =:= auto ->
    start()
  end
```